src/utils/nlpUtils/OPLparser.py

```diff
@@ -16,113 +16,205 @@
+    self.OPLtextParser()
+    self.OPLparser()
+
+    
+  def OPLentityParser(self):
+    '''
+    This method extracts all the form and function entities out of the OPL html file and it puts them in two separate lists
+    @in: filename, file name containing the OPL text
+    @out: objectList, list, list of form elements contained in the OPL file
+    @out: processList, list, list of function elements contained in the OPL file
+    '''
+    with open(self.filename) as fp:
+      soup = BeautifulSoup(fp, "html.parser")
+      elements = soup.find("font").findChildren()
+      for element in elements:
+        if element.has_attr('color'):
           elem = element.string.replace("\n", " ")
-          if elem not in objectList:
-            objectList.append(elem)
-        elif element['color'] == '#000078':
-          elem = element.string.replace("\n", " ")
-          if elem not in processList:
-            processList.append(elem)
-  return objectList, processList
+          elem = checkAcronym(elem)
+          if elem[1] is not None:            
+            self.acronyms[elem[1]] = elem[0]
+          elem = elem[0]
+          if element['color'] == '#006d00': 
+            if elem.lower() not in self.objectList:
+              self.objectList.append(elem.lower())
+          elif element['color'] == '#000078':
+            if elem.lower() not in self.processList:
+              self.processList.append(elem.lower())   
+
+
+  def OPLtextParser(self):
+    '''
+    This method extracts all the sentences out of the OPL html file and it puts them in a list
+    @in: filename, file name containing the OPL text
+    @out: objectList, sentences, list of sentenced contained in the OPL file
+    '''
+    objects = {}
+    functions = {}
+    with open(self.filename) as fp:
+      soup = BeautifulSoup(fp, "html.parser")
+
+      # kill all script and style elements
+      for script in soup(["script", "style"]):
+          script.extract()    # rip it out
+      # get text
+      text = soup.get_text()
+      # break into lines and remove leading and trailing space on each
+      lines = (line.strip() for line in text.splitlines())
+      # break multi-headlines into a line each
+      chunks = (phrase.strip() for line in lines for phrase in line.split("  "))
+      # drop blank lines
+      text = '\n'.join(chunk for chunk in chunks if chunk)
+      self.sentences = text.split(".")
+
+      for index, sentence in enumerate(self.sentences):
+        self.sentences[index] = self.sentences[index].replace("\n", " ")
+        self.sentences[index] = self.sentences[index].replace("\xa0", "")
+        self.sentences[index] = self.sentences[index].lstrip().lower()
+
+      self.sentences.remove('')
+
+
+  def OPLparser(self):
+    '''
+    This method translates all the sentences create a graph structure
+    @in: filename: file name containing the OPL text
+    @out: objectList: sentences, list of sentenced contained in the OPL file
```

Comment:
please update the docstring for this function, no input and no output for this function